corehq/apps/export/views.py

```diff
@@ -632,19 +632,21 @@ def _get_download_task(self, in_data):
             )
         return download
 
+    def _get_and_rebuild_export_schema(self, export_id):
+        export_object = self.get_export_schema(self.domain, export_id)
+        export_object.update_schema()
+        return export_object
+
     def _get_bulk_download_task(self, export_specs, export_filter):
+        for export_spec in export_specs:
+            export_id = export_spec['export_id']
+            self._get_and_rebuild_export_schema(export_id)
         export_helper = CustomBulkExportHelper(domain=self.domain)
         return export_helper.get_download_task(export_specs, export_filter)
 
     def _get_single_export_download_task(self, export_spec, export_filter, max_column_size=2000):
-        try:
-            export_data = export_spec
-            export_object = self.get_export_schema(self.domain, export_data['export_id'])
-        except (KeyError, IndexError):
-            raise ExportAsyncException(
-                _("You need to pass a list of at least one export schema.")
-            )
-        export_object.update_schema()
+        export_id = export_spec['export_id']
+        export_object = self._get_and_rebuild_export_schema(export_id)
```

Comment:
Well, I think the individual exports were just starting by rebuilding the schemas no matter what, and this PR changes the bulk export to do that too. It's not the most efficient thing (in bulk exports, it does this solely to rebuild them in case they need it, and then just continues doing what it was doing before), but I don't think it'll significantly change how much time is spent in a bulk export.